pkg/scheduler/scheduler.go

```diff
@@ -546,7 +546,7 @@ func (sched *Scheduler) scheduleOne() {
 	// bind the pod to its host asynchronously (we can do this b/c of the assumption step above).
 	go func() {
 		// Bind volumes first before Pod
-		if !allBound {
+		if !allBound && utilfeature.DefaultFeatureGate.Enabled(features.VolumeScheduling) {
```

Comment:
Maybe handle it in assumeVolumes phase will be better.

Reply:
done. PTAL